src/components/Legend.jsx

```diff
@@ -24,211 +25,159 @@ const LegendText = styled.div`
-        <line x1="0" y1="15" x2="30" y2="15" stroke={featureColors.protectedBikeLane} strokeWidth="8" />
-        <line x1="0" y1="15" x2="30" y2="15" stroke="#FFFFFF" strokeWidth="4" strokeDasharray="12,5" />
-      </svg>
-    ),
-    label: "Protected Bike Lane - Design or Construction",
-  },
-  {
-    src: (
-      <svg width="30" height="30" xmlns="http://www.w3.org/2000/svg">
-        <line x1="0" y1="15" x2="30" y2="15" stroke={featureColors.bikeLane} strokeWidth="8" />
-        <line x1="0" y1="15" x2="30" y2="15" stroke="#FFFFFF" strokeWidth="4" />
-      </svg>
-    ),
-    label: "Bike Lane and Sidewalk",
-  },
-  {
-    src: (
-      <svg width="30" height="30" xmlns="http://www.w3.org/2000/svg">
-        <line x1="0" y1="15" x2="30" y2="15" stroke={featureColors.bikeLane} strokeWidth="8" />
-        <line x1="0" y1="15" x2="30" y2="15" stroke="#FFFFFF" strokeWidth="4" strokeDasharray="12,5" />
-      </svg>
-    ),
-    label: "Bike Lane - Design or Construction",
-  },
-  {
-    src: (
-      <svg width="30" height="30" xmlns="http://www.w3.org/2000/svg">
-        <line x1="0" y1="15" x2="30" y2="15" stroke={featureColors.sharedStreet} strokeWidth="8" />
-        <line x1="0" y1="15" x2="30" y2="15" stroke="#FFFFFF" strokeWidth="4" />
-      </svg>
-    ),
-    label: "Shared Street - Urban",
-  },
-  {
-    src: (
-      <svg width="30" height="30" xmlns="http://www.w3.org/2000/svg">
-        <line x1="0" y1="15" x2="30" y2="15" stroke={featureColors.sharedStreet} strokeWidth="8" />
-      </svg>
-    ),
-    label: "Shared Street - Suburban",
-  },
-  {
-    src: (
-      <svg width="30" height="30" xmlns="http://www.w3.org/2000/svg">
-        <line x1="0" y1="15" x2="30" y2="15" stroke={featureColors.sharedStreet} strokeWidth="8" strokeDasharray="12, 5" />
-      </svg>
-    ),
-    label: "Shared Street - Envisioned",
-  },
-
-  {
-    src: (
-      <svg width="30" height="30" xmlns="http://www.w3.org/2000/svg">
-        <line x1="0" y1="15" x2="30" y2="15" stroke="#888888" strokeWidth="8" />
-        <line x1="0" y1="15" x2="30" y2="15" stroke={featureColors.Gap} strokeWidth="7" />
-      </svg>
-    ),
-    label: "Gap - Facility Type TBD",
-  },
-  {
-    src: (
-      <svg width="30" height="30" xmlns="http://www.w3.org/2000/svg">
-        <line x1="0" y1="15" x2="30" y2="15" stroke={featureColors.footTrail} strokeWidth="8" />
-      </svg>
-    ),
-    label: "Foot Trail - Existing",
-  },
-  {
-    src: (
-      <svg width="30" height="30" xmlns="http://www.w3.org/2000/svg">
-        <line x1="0" y1="15" x2="30" y2="15" stroke={featureColors.footTrail} strokeWidth="8" strokeDasharray="12, 5" />
-      </svg>
-    ),
-    label: "Foot Trail - Envisioned",
-  },
-];
+
 
 const Legend = ({ selectedTab = "landlines", selectedType, projectList = [], selectedProject = null, handleProjectSelect = () => {} }) => {
+  const statusStyles = [
+    {
+      label: "Existing",
+      style: (color) => (
```

Comment:
`color` seems to be an unused variable here; can we just return the svg element directly instead of making this a function call?